languages/fsharp/docs/examples/new-concept-exercise-strings.md

```diff
@@ -0,0 +1,164 @@
+This issue describes how to implement the `strings` concept exercise for the F# track.
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Strings in [F#][docs.microsoft.com-string].
+
+## Learning objectives
+
+- Know of the existence of the `string` type.
+- Know how to create a string.
+- Know of the existence of `string` methods and `String` module function
+- Know some basic string methods (like finding the index of a character at a position, or returning a part the string).
+- Know how to do some basic string formatting (using `+` and `sprintf`).
+- Know that strings are immutable.
+
+## Out of scope
+
+- Strings can be enumerated.
+- String slicing.
+- Triple-quoted strings.
+- Double-backtick strings.
+- Advanced string formatting (combining types e.g.).
```

Comment:
Is this `e.g.` missing something afterwards?

Reply:
Doh. Fixed.